em++.py

```diff
@@ -1,19 +1,17 @@
-#!/usr/bin/env python
```

Comment:
why is em++.py special among the tools? seems like it's getting different treatment in this PR than the others and it's not obvious why.

Reply:
Because em++ was using the python_selector in a slightly different way.  It was using it to redirect to emcc.py as well as to choose the python version.

With this change, once python is running we assume its the correct version, we never try to re-invoke python.  So this change means that em++ now runs emcc directly, without re-invoking python.